charts/rpe-send-letter-service/values.yaml

```diff
@@ -1,37 +1,58 @@
 java:
   applicationPort: 8485
+  aadIdentityName: send-letter
   keyVaults:
     rpe-send-letter:
-      resourceGroup: rpe-send-letter-service
       secrets:
         - ftp-user
         - ftp-private-key
         - ftp-public-key
+        - upload-summary-report-recipients
+        - send-letter-service-POSTGRES-PASS
+        - encryption-public-key
+        - app-insights-instrumentation-key
+        - reports-email-password
+        - reports-email-username
   environment:
     # db
-    LETTER_TRACKING_DB_HOST: "{{ .Release.Name }}-postgresql"
-    LETTER_TRACKING_DB_NAME: "{{ .Values.postgresql.postgresqlDatabase}}"
-    LETTER_TRACKING_DB_USER_NAME: "{{ .Values.postgresql.postgresqlUsername}}"
-    LETTER_TRACKING_DB_PASSWORD: "{{ .Values.postgresql.postgresqlPassword}}"
+    LETTER_TRACKING_DB_HOST: "rpe-send-letter-service-db-{{ .Values.global.environment }}.postgres.database.azure.com"
+    LETTER_TRACKING_DB_PORT: "5432"
+    LETTER_TRACKING_DB_NAME: "send_letter"
+    LETTER_TRACKING_DB_USER_NAME: "send_letter@rpe-send-letter-service-db-{{ .Values.global.environment }}"
```

Comment:
I still don't like hardcoded values here. Would it make sense if we load from vault and move to bootstrap as well?